### DISCUSSION TITLE
Add viewing and editing video device information to virtual machine information

### DISCUSSION THREAD

**Discussion by Shotaro-Kawaguchi** [cockpit-project/cockpit-machines#1670] [Ideas]
Page: Virtual Machines

Description:

I want to add the ability to view and edit video devices to the virtual machine details.
I'm assuming something like the attached image.

Currently, cockpit-machines can check the usage of resources such as CPU, memory, and disk as virtual machine information.
However, we cannot view or edit the video device information.

Adding video device viewing and editing to virtual machine information provides the following benefits:
Determined the specifications of the video device assigned to the virtual machine.
Enable to set VNC password.

I am planning to make a patch, if anyone has any comments, please let me know.
![VideoDevices](https://github.com/cockpit-project/cockpit-machines/assets/171650389/54917089-d74f-4c44-9565-fc0ee3529c10)

**Comment by JeffWDH** [comment]
The ability to choose the video device driver would be nice as well. The Cockpit default of virtio can causes issues with Linux and you can end up seeing an error in the VNC viewer that says 'display output is not active'. Changing the driver to vga solves this, but you need to do it either in virt-manager or via virsh currently.
